board/drivers/fake_siren.h

```diff
@@ -2,7 +2,36 @@
 
 #define CODEC_I2C_ADDR 0x10
 
-void fake_siren_init(void);
+// 1Vpp sine wave with 1V offset
+// cppcheck-suppress misra-c2012-8.9
```

Comment:
let's try to not any suppressions. doesn't look like there's a fundamental reason to not keep the LUT defined in the init function?